sdks/python/apache_beam/io/textio.py

```diff
@@ -21,7 +21,7 @@
 from __future__ import absolute_import
 import logging
 
-from apache_beam import coders
+from apache_beam.coders import coders
```

Comment:
What is the guidance now for usage of the `Coder` class?  Are users expected to do something like this?

```python
class MyCoder(beam.coders.coders.Coder):
  ...
```
Did you also mean to change `coders/__init__.py`, or is the simpler form (`beam.coders.Coder`) still permitted / recommended in user code?  (CC: @sb2nov)